### PR TITLE
Fix typo in the example config

### PR DESCRIPTION
**Pull request by a13xg0**
I think, it is related to the #2 issue

### CODE REVIEW

**Comment by mrts** [comment]
Excellent catch, thank you! It completely baffles me how this can even work in our system, but it does :) - maybe someone fixed it locally.

**Comment by ghost** [comment]
End

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Clone the repository, mount its directory as a volume into
     myapp-postgresql:
         image: postgres:9.6.2
         volumes:
-            - ../docker-posgresql-multiple-databases:/docker-entrypoint-initdb.d
+            - ../docker-postgresql-multiple-databases:/docker-entrypoint-initdb.d
```
Comment (ghost) [inline README.md:28]: End

File: README.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ Clone the repository, mount its directory as a volume into
     myapp-postgresql:
         image: postgres:9.6.2
         volumes:
-            - ../docker-posgresql-multiple-databases:/docker-entrypoint-initdb.d
```
Comment (ghost) [inline README.md:28]: End

File: README.md [hunk 3]
```diff
@@ -25,7 +25,7 @@ Clone the repository, mount its directory as a volume into
...
+            - ../docker-postgresql-multiple-databases:/docker-entrypoint-initdb.d
         environment:
             - POSTGRES_MULTIPLE_DATABASES=db1,db2
```
Comment (ghost) [inline README.md:30]: End

File: README.md [hunk 4]
```diff
@@ -25,7 +25,7 @@ Clone the repository, mount its directory as a volume into
...
             - POSTGRES_USER=myapp
```
Comment (ghost) [inline README.md:31]: Flase

File: README.md (hunk 2 above)
Comment (ghost) [inline README.md:28]: End

**Pull request merged**